### DISCUSSION TITLE
一键更新相关

### DISCUSSION THREAD

**Discussion by 2607627233** [dyphire/mpv-config#49] [Q&A] [answered]
"git":"https://github.com/dyphire/uosc",
    "whitelist":"readme%.md$|src/uosc/",
    "dest":"~~/scripts/uosc",
    "branch":"dev"
这个放置的路径不对啊 ，是更新之后还要手动放置吗

**Comment by dyphire** [comment, marked as answer]
由提交 https://github.com/dyphire/mpv-config/commit/d0b8fef7f1ef593ec34e98cc8d2693a1390a3fc0 移除
uosc 上游仓库的文件结构的原因，更新脚本无法正常处理。只能自己手动更新
